ansible-ipi-install/roles/node-prep/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
while I'd like to see tags a lot more places, would especially be nice here to controll execution of the top level tasks.

Reply:
Feature enhancement #132